resources/sass/layout/_header.scss

```diff
@@ -0,0 +1,157 @@
+    nav {
+        -webkit-box-shadow: 0 8px 6px -6px rgba(0, 0, 0, .15);
+        -moz-box-shadow: 0 8px 6px -6px rgba(0, 0, 0, .15);
+        box-shadow: 0 8px 6px -6px rgba(0, 0, 0, .15);
+    }
+    .row-header {
+        @include div-row;
+        text-align: center;
+        ul>li {
+            margin-left: 20px;
+            a {
+                text-decoration: none;
+                color: #787677;
+                font-weight: 600;
+            }
+        }
+        .logo {
+            flex: 0;
+            height: 70px;
+            width: auto;
+        }
+        .main-nav {
+            flex: 1;
+            display: flex;
+            align-items: center;
+            margin-left: 11%;
+            margin-top: 3%;
+        }
+        .super-header {
+            flex: 0;
+            font-size: 14px;
+            .cart-section {
+                margin-top: 21%;
+                display: flex;
+                background-color: #eeeeee;
+                width: 100px;
+                align-items: center;
+                justify-content: center;
+                .img {
+                    width: 40px;
+                    height: 40px;
+                }
+            }
+        }
+    }
+}
+
+.header-nav {
+    display: flex;
+    list-style: none;
+}
+
+.show {
+    display: none;
+}
+
+.hide {
+    display: block;
+}
+
+.hide-menu {
+    display: inline-block;
+}
+
+.w3-teal,
+.w3-hover-teal:hover {
+    color: #fff !important;
+    background-color: #0e0e0e !important;
+}
+
+
+/* for ipad */
+
+@media screen and (max-width: 480px) {
+    .show {
+        display: block !important;
+    }
+    .hide-menu {
+        display: none !important;
+    }
+    .hide {
+        display: none !important;
+    }
+    .header-nav {
+        display: none !important;
+    }
+    .row-header {
+        @include div-row;
+        text-align: center;
+        height: 45px;
+        /* justify-content: center; */
+    }
+    .logo_img {
+        width: 25%;
+        padding: 2%;
+    }
+    .cart-section {
+        margin-top: -7% !important;
+    }
+    body {
+        padding: 2%;
+    }
+    .w3-teal,
+    .w3-hover-teal:hover {
+        color: #fff !important;
+        background-color: #0e0e0e !important;
+    }
+    .mobile_menu {
+        margin-left: 55%;
+        color: #8a8a8a;
+        font-size: 18px;
+        font-weight: 400;
+        a {
+            text-decoration: none;
+        }
+        .w3-sidebar {
+            height: auto !important;
+            width: 200px;
+            background-color: #eaeaea !important;
+            position: fixed !important;
+            z-index: 1;
+            overflow: auto;
+        }
+    }
+}
+
+
+/*tablet */
+
+@media screen and (max-width: 767px) {
+    .show {
+        display: none;
+    }
+    .super-header {
+        padding-right: 1%;
+    }
+    .cart-section {
+        margin-left: 16%;
+    }
+}
+
+
+/*tablet  landscape*/
+
+@media screen and (max-width: 1023px) {
```

Comment:
There media queries should contain the scope of this module. Think of what will happen to it when all of this CSS join one CSS file